scss/_alerts.scss

```diff
@@ -90,3 +90,17 @@
   margin-bottom: 0;
   line-height: 1.5;
 }
+
+.warning {
+  padding: 0.5em;
+  margin-bottom: 0.8em;
+  font-weight: bold;
+  background-color: #fffccc;
+}
+
+.error_box {
+  padding: 1em;
+  font-weight: bold;
+  background-color: #ffebe8;
+  border: 1px solid #dd3c10;
+}
```

Comment:
They're already in there https://github.com/primer/primer/blob/master/scss/_utility.scss#L74-L86 just moving them where I thought would make more sense than `utility`